### PR TITLE
Fixes to management API documentation

### PR DESCRIPTION
**Pull request by vrakesh**
*Issue #, if available:*

*Description of changes:*

**To run CI tests on your changes refer [README.md](https://github.com/awslabs/mxnet-model-server/blob/master/ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/management_api.md [hunk 1]
```diff
@@ -74,8 +74,8 @@ curl -v -X POST "http://localhost:8081/models?url=https%3A%2F%2Fs3.amazonaws.com
 ### Scale workers
 
 `PUT /models/{model_name}`
-* min_worker - (optional) the minimum number of worker processes. MMS will try to maintain this minimum for specified model. The default value is `1`.
-* max_worker - (optional) the maximum number of worker processes. MMS will make no more that this number of workers for the specified model. The default is the same as the setting for `min_worker`.
+* min_workers - (optional) the minimum number of worker processes. MMS will try to maintain this minimum for specified model. The default value is `1`.
```
Comment (vdantu) [inline docs/management_api.md]: Is this correct? Isn't it singular currently?

File: docs/management_api.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ User may want to create workers while register, creating initial workers may tak
 The asynchronous call will return before trying to create workers with HTTP code 202:
 
 ```bash
-curl -v -X POST "http://localhost:8081/models?url=https%3A%2F%2Fs3.amazonaws.com%2Fmodel-server%2Fmodels%2Fsqueezenet_v1.1%2Fsqueezenet_v1.1.model&initial_workers=1&synchronous=true"
+curl -v -X POST "http://localhost:8081/models?url=https://s3.amazonaws.com/model-server/model_archive_1.0/squeezenet_v1.1.marl&initial_workers=1&synchronous=false"
```
Comment (frankfliu) [inline docs/management_api.md]: move initial_workers=1&synchronous=false before url

File: docs/management_api.md (hunk 1 above)
Comment (frankfliu) [inline docs/management_api.md]: min_worker

File: docs/management_api.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ User may want to create workers while register, creating initial workers may tak
 The asynchronous call will return before trying to create workers with HTTP code 202:
 
 ```bash
-curl -v -X POST "http://localhost:8081/models?url=https%3A%2F%2Fs3.amazonaws.com%2Fmodel-server%2Fmodels%2Fsqueezenet_v1.1%2Fsqueezenet_v1.1.model&initial_workers=1&synchronous=true"
+curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=false&url=https%3A%2F%2Fs3.amazonaws.com%2Fmodel-server%2Fmodels%2Fsqueezenet_v1.1%2Fsqueezenet_v1.1.model"
```
Comment (vdantu) [inline docs/management_api.md]: Why do we need synchronous=false?

**Pull request merged**